kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KopBrokerLookupManager.java

```diff
@@ -98,7 +100,11 @@ public KopBrokerLookupManager(KafkaServiceConfiguration conf, PulsarService puls
         if (log.isDebugEnabled()) {
             log.debug("Handle Lookup for topic {}", topicName);
         }
-        return LOOKUP_CACHE.computeIfAbsent(topicName, this::lookupBroker);
```

Comment:
I can't understand this change.
can you please explain ?

Reply:
If we cache all topics' lookup result, we will meet the issue here 
https://github.com/streamnative/kop/issues/1111#issuecomment-1066583891

Reply:
but `computeIfAbsent` is like 
```
if (map.containsKey()) {
 return map.get();
}
Result r = compute();
put(r)
return r;
```

so it is basically the same code, but without concurrency guarantees